packages/ui/src/Tooltip.module.scss

```diff
@@ -1,26 +1,135 @@
 @use '../styles/constants' as c;
+@use '../styles/helpers' as h;
 
-.wrapper {
-  max-width: c.$tooltipMaxWidth;
-  word-wrap: break-word;
+.tooltip-sr {
+  @include h.visuallyHidden;
 }
 
 .overlay {
-  // TODO: Update colors. Ask Pawel.
-  color: #fff;
+  @include c.typographyBodySmall;
+
+  position: relative;
+  max-width: c.$tooltipMaxWidth;
+  padding: c.$grid * 5;
+  background: c.$colorNeutralLighter;
+  border: c.$borderWidth solid c.$colorNeutralLight;
+  border-radius: c.$borderRadius;
+  font-size: inherit;
```

Comment:
Why do you think we need to inherit the font-size? I figured it would be the same for all tooltips